balancer-js/src/lib/utils/poolHelper.ts

```diff
@@ -8,26 +8,32 @@ import {
 } from '@/lib/utils/solidityMaths';
 import { AssetHelpers } from '@/lib/utils/assetHelpers';
 
-const AMP_PRECISION = 3; // number of decimals -> precision 1000
+export const AMP_PRECISION = 3; // number of decimals -> precision 1000
 
-interface ParsedPoolInfo {
-  parsedTokens: string[];
-  parsedDecimals: string[];
-  parsedBalances: string[];
-  parsedWeights: string[];
-  parsedPriceRates: string[];
-  parsedAmp: string;
-  parsedTotalShares: string;
-  parsedSwapFee: string;
-  upScaledBalances: string[];
+type ParsedPoolInfo = {
+  athRateProduct: string;
```

Comment:
Is this used anywhere?

Reply:
Yeah - these were copy/pasted from the changes that Luiz made on his branches. The idea was to make it easier for him to handle merge conflicts, but looking at it now, it likely makes more sense to remove them and let him add them with the proper types. I'll do that 👍